R/db-odbc-mssql.R

```diff
@@ -47,6 +47,7 @@
       atan2         = sql_prefix("ATN2"),
       ceil          = sql_prefix("CEILING"),
       ceiling       = sql_prefix("CEILING"),
+      sd            = sql_prefix("STDEV"),
```

Comment:
I just realised that `sd` shouldn't appear here - it's a summary or window function, not a scalar function.